lib/worker.js

```diff
@@ -111,6 +112,51 @@ Worker.prototype._reportError = function (err) {
 }
 
 /**
+ * Helper function for creating monitor-dog events tags
+ * `queue` is the only mandatory tag.
+ * Few tags would be created depednding on the queue name
+ * If queueName use `.` as delimiter e.x. `10.0.0.20.api.github.push` then
+ * following tags would be created:
+ * - token0: 'push'
+ * - token1: 'github.push'
+ * - token2: 'api.github.push'
+ * - token3: '10.0.0.20.api.github.push'
+ * @private
+ * @returns {Object} tags as Object {queue: 'docker.event.publish'}
+ */
+Worker.prototype._eventTags = function () {
+  var tokens = this.queue.split('.').reverse()
+  var lastToken = ''
+  var tags = tokens.reduce(function (acc, currentValue, currentIndex) {
+    var key = 'token' + currentIndex
+    var newToken = (currentIndex === 0 ? currentValue : currentValue + '.' + lastToken)
+    acc[key] = newToken
+    lastToken = newToken
+    return acc
+  }, {})
+  tags.queue = this.queue
+  return tags
+}
+
+/**
+ * Helper function calling `monitor.increment`.
+ * Monitor wouldn't be called if `process.env.WORKER_MONITOR_DISABLED` set.
+ * @param {string} eventName name to be reported into the datadog
+ * @param {object} [extraTags] extra tags to be send with the event
+ * @private
+ */
+Worker.prototype._inc = function (eventName, extraTags) {
```

Comment:
Can we rename this function? `_inc` is maddeningly unhelpful when I was reading it below. `_incMonitor`, maybe?